modules/ROOT/pages/db-util.adoc

```diff
@@ -4,66 +4,14 @@ include::_attributes.adoc[]
 endif::[]
 :page-aliases: munit-database-server.adoc
 
-== Overview
-
 To test a piece of code that connects with a database server, you must install a database server in your local environment to run the tests. You could also have a dedicated external database server for testing, but your Maven project would not be portable -- a third party would need to install the testing database server to compile your project.
 
 To make it easier for you to test database connections, MUnit allows you to implement a database server in your local environment.
 
-== Example
-
-. Assume that you want to test the following Mule application:
-+
-[source,xml,linenums]
-----
-<!-- Local Database Configuration -->
-<db:config name="Database_Config">
-    <db:my-sql-connection host="localhost" port="1234" />
-</db:config>
-<!-- Properties according to the environment -->
-<configuration-properties file="db.properties" />
-
-<flow name="selectFlow" >
-    <!-- Perform a query based on a value passed by a variable -->
-    <db:select config-ref="${db.config}">
-        <db:sql >SELECT * FROM jobtitlelookup WHERE jobtitleid = :id</db:sql>
-        <db:input-parameters ><![CDATA[#[id : vars.jobid]]]></db:input-parameters>
-    </db:select>
-    <!-- Set two conditions depending on the query result -->
-    <choice>
-        <!-- If there is one or more values resulting from the query set those values as the payload -->
-        <when expression="#[sizeOf(payload) > 0]">
-            <set-payload value="#[payload[0].jobtitle]"/>
-        </when>
-        <!-- If the query throws no results, inform it in the payload  -->
-        <otherwise>
-            <set-payload value="#['No job title for $(vars.jobid) was found']" />
-        </otherwise>
-    </choice>
-
-    <!-- Finally, pass the payload content as a Flow variable  -->
-    <set-variable value="#[payload]" variableName="job" />
-</flow>
-----
-. The file `db.properties` in `src/main/resources` has the following content:
-+
-[source]
-----
-db.config=Database_Config
-----
-. You need to create a test that:
-.. Contains a Database Server with a `jobtitlelookup` table.
-.. Provides a valid Database Structure for the Database connector to perform its' query.
-.. Passes a value for the `jobid` variable that the Database connector uses.
-
-=== Installing MUnit DB Server Module
+== Installing the MUnit DB Server Module
```

Comment:
DITTO edits made to previous Lines 51-99.
Is this the main "home" for this topic? If so, can we xref to THIS topic as a prerequisite to Configuring Database Server Util in an MUnit Test, rather than as a full subtask?